src/commons/BrowserUtils.ts

```diff
@@ -182,12 +188,16 @@ export namespace BrowserUtils {
    * @param url expected current url
    */
   export function expectCurrentUrl(url: string): void {
+    Reporter.debug(`Raw URL: [${url}]`);
     const expectedUrl: string = normalizeUrl(url);
-    Reporter.debug(`Wait for URL to be , ${expectedUrl}`);
+    Reporter.debug(`Wait for URL to be , [${expectedUrl}]`);
     let currentUrl: string;
     browser.waitUntil(
       (): boolean => {
         currentUrl = normalizeUrl(getUrl());
+        Reporter.debug(
```

Comment:
this is in waitUntil 'loop'. it will be printed each iteration. Is it what you wanted to add?

Reply:
Question should be if there is a need for the wait until here...